lib/loader/index.js

```diff
@@ -259,76 +259,40 @@ function postInstantiate(extendedExports, instance) {
   function getTypedArrayView(Type, alignLog2, ptr) {
     const buffer = memory.buffer;
     const U32 = new Uint32Array(buffer);
-    const bufPtr = U32[ptr + ARRAYBUFFERVIEW_DATASTART_OFFSET >>> 2];
+    const id = U32[ptr + ID_OFFSET >>> 2];
+    const info = getInfo(id);
```

Comment:
@MaxGraey Are you ok with this? Iirc this helper originated from trying to avoid every inch of overhead, including not having to read RTTI with `getInfo`.

Reply:
Oh. Yeah. Should we just revert back to what I had before? I really don't mind having a larger surface area of functions like I had before.